com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeAsset.cs

```diff
@@ -50,6 +75,86 @@ public string GetSerializedFullPath()
             return m_AssetFullPath;
         }
 
+        internal void ResolveCells()
```

Comment:
As we discussed the stream should be simply float based

Reply:
Done.